examples/purchaseTesterTypescript/metro.config.js

```diff
@@ -5,13 +5,36 @@
  * @format
  */
 
-module.exports = {
-  transformer: {
-    getTransformOptions: async () => ({
-      transform: {
-        experimentalImportSupport: false,
-        inlineRequires: true,
-      },
-    }),
-  },
-};
+// IMPORTANT
+// Make sure `npm install` is run in this refernced package's directory
```

Comment:
```suggestion
// Make sure `npm install` is run in this referenced package's directory
```

:)

Reply:
Typos are going to be my legacy at RevenueCat but I'll fix it this one time 🙃